docker/Dockerfile

```diff
@@ -1,12 +1,176 @@
-FROM debian:11
-MAINTAINER Pangeanic <info@pangeanic.com>
+FROM debian:bullseye-slim
+
+MAINTAINER Hakan Baysal <hakanbysl@gmail.com>
```

Comment:
@higherstar I’m afraid that there has been a misunderstanding. When I talked about changing the base image `python:3.9.17-bullseye`, I did not mean `debian:bullseye-slim`.

To use the base image `python:3.9.17-bullseye`, the Dockerfile must start with the following line:
```Dockerfile
FROM python:3.9.17-bullseye
```